<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingTab</class>
 <widget class="QWidget" name="TrainingTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="lay" stretch="1,0">
   <item>
    <layout class="QFormLayout" name="settings_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="1">
      <widget class="QWidget" name="count_menu" native="true">
       <layout class="QHBoxLayout" name="count_layout">
        <item>
         <widget class="QSlider" name="count_slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="count_box">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="type_menu_label">
       <property name="text">
        <string>Типы вопросов</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="count_menu_label">
       <property name="text">
        <string>Количество вопросов</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="type_menu">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="type_checkbox">
          <property name="text">
           <string>type name</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="difficulty_menu_label">
       <property name="text">
        <string>Сложность</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="difficulty_menu">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="difficulty_checkbox">
          <property name="text">
           <string>difficulty number</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_layout" stretch="1,2,1">
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="styleSheet">
        <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="button_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="styleSheet">
        <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 2px;
border: 2px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 2px solid #26003d;
</string>
       </property>
       <property name="text">
        <string>Начать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
